docs/UserGuide.md

```diff
@@ -66,70 +74,95 @@ Choose a topic from the table of contents to get started on your Wishful Shrinki
-* `CALORIES` **must be a positive integer** e.g. 100, 200, 300...
+<div markdown="span" class="alert alert-primary">:memo: **Note:**
+`QUANTITY` is separated with a mandatory space before `-` and after the dash, only accepts alphanumeric characters
+, forward slashes and full stops.
+</div>
+<div markdown="span" class="alert alert-primary">:memo: **Note:**
+Multiple ingredients are separated by `,`.
+</div> 
+
+* `CALORIES` **must be a positive integer** e.g. 150, 200...
 * `IMAGE` can be local path e.g. images/healthy1.jpg or url e.g. https://vaya.in/recipes/wp-content/uploads/2018/06/Club-sandwich.jpg
-   * Note: If you don't input the image field, Wishful Shrinking will add a default image to the recipe.
-   * Note: If you want to use an image that has been saved in your computer, you have to specify the **absolute path** of the image, and add file:// in front of the file path. Then, replace IMAGE in img/IMAGE with the file path of your image, e.g. img/file:///D:/myimages/wishful/data/myimage.png.
-   * Note: If you want to use a url as the image, make sure that you are connected to the internet on your computer, and that the url is the **image address** (starting with https:// and ending with .jpg/jpeg/png). The image will only be downloaded into Wishful Shrinking's data folder if there is internet connection.
-   * Note: If you input an invalid file path, input an invalid url, or input a url while there is no internet,  Wishful Shrinking will replace the image with a default image. 
-   * Wishful Shrinking also has a few sample images for you to use if you don't have any images on hand. Below is the table of sample images and their file paths. To use the sample images, replace IMAGE in img/IMAGE with the file path, e.g. img/images/healthy1.jpg.<br>
+    * If you don't input the image field, Wishful Shrinking will add a default image to the recipe.
+    * If you want to use an image that has been saved in your computer, you have to specify the **absolute path** of the image, and add file:// in front of the file path. Then, replace IMAGE in img/IMAGE with the file path of your image, e.g. img/file:///D:/myimages/wishful/data/myimage.png.
+    * If you want to use a url as the image, make sure that you are connected to the internet on your computer, and that the url is the **image address** (starting with https:// and ending with .jpg/jpeg/png). The image will only be downloaded into Wishful Shrinking's data folder if there is internet connection.
+    * If you input an invalid file path, input an invalid url, or input a url while there is no internet,  Wishful Shrinking will replace the image with a default image. 
+    * Wishful Shrinking also has a few sample images for you to use if you don't have any images on hand. Below is the table of sample images and their file paths. To use the sample images, replace IMAGE in img/IMAGE with the file path, e.g. img/images/healthy1.jpg.<br>
```

Comment:
I think we can remove the 'also' in this sentence since is a bullet point.

Reply:
you're right ok